dnf-behave-tests/common/output.py

```diff
@@ -66,6 +89,38 @@ def then_stdout_is_empty(context):
     raise AssertionError("Stdout is not empty, it contains: %s" % context.cmd_stdout)
 
 
+@behave.then("stdout is help")
+def then_stdout_is_help(context):
+    expected = context.text.format(context=context).rstrip().split('\n')
+    found = context.cmd_stdout.rstrip().split('\n')
+
+    if found == [""]:
+        found = []
+
+    dnf5_mode = hasattr(context, "dnf5_mode") and context.dnf5_mode
+    clean_expected, clean_found = handle_help(expected, found, dnf5_mode)
```

Comment:
The `handle_help()` method doesn't exist, and the `stdout is help` step (or its dnf4/dnf5 variants) is never used...